etc/deltaDebug.py

```diff
@@ -110,7 +116,6 @@ def debug(self):
 
         while (True):
             err = None
-            self.n = 2  # Initial Number of cuts
```

Comment:
You are changing the default from 2 to 1 - why?

Reply:
good catch. I have this set to draft until I can test it....